bitcoinspv/clients/sui/utils.go

```diff
@@ -46,3 +47,48 @@ func BlockHeaderToHex(header wire.BlockHeader) (string, error) {
 func BlockHashToHex(hash chainhash.Hash) string {
 	return "0x" + hex.EncodeToString(hash.CloneBytes())
 }
+
+type bcsEncode []byte
+
+func getBCSResult(res *suiclient.DevInspectTransactionBlockResponse) []bcsEncode {
+	bcsEncode := make([]bcsEncode, len(res.Results[0].ReturnValues))
+
+	for i, item := range res.Results[0].ReturnValues {
+		var b []byte
+		// TODO: Breakdown to simple term
+		c := item.([]interface{})[0].([]interface{})
+		b = make([]byte, len(c))
+
+		for i, v := range c {
+			b[i] = byte(v.(float64))
+		}
+		bcsEncode[i] = b
+	}
+	return bcsEncode
+}
+
+// BlockHeader is block header
+type BlockHeader struct {
+	Internal []uint8
+}
+
+// LightBlock is light block
+// TODO: fix lint
+//
+//nolint:govet
+type LightBlock struct {
```

Comment:
Light client and relayer should have an agreement about the data structure they are using. 
Light Block <> Block Info 
BlockHeader <> wire.Header. 
We should create another task for this.

Reply:
cc: @robert-zaremba @sczembor